ds/fact_evaluation7.py

```diff
@@ -35,13 +36,38 @@ def extract(data_src, gzip=False, decimal='.'):
     if gzip:
         compress = 'gzip'
     compress = 'infer'
-    cols = ['NU_ANO', 'CO_IES', 'CO_GRUPO', 'CO_CURSO', 'CO_MODALIDADE',
-            'CO_MUNIC_CURSO', 'CO_UF_CURSO', 'NU_IDADE', 'TP_SEXO',
-            'TP_INSCRICAO', 'TP_PRES', 'NT_GER', 'NT_FG', 'NT_CE',]
+    
+    cols = ['NU_ANO', 'CO_IES', 'CO_GRUPO', 'CO_CURSO', 'CO_MODALIDADE', \
+            'CO_MUNIC_CURSO', 'CO_UF_CURSO', 'NU_IDADE', 'TP_SEXO', \
+            'TP_INSCRICAO', 'TP_PRES', 'NT_GER', 'NT_FG', 'NT_CE']
+    
+    if (data_src == './datasrc/ENADE_2004.csv.gz') or (data_src == './datasrc/ENADE_2005.csv.gz') or \
+       (data_src == './datasrc/ENADE_2006.csv.gz') or (data_src == './datasrc/ENADE_2007.csv.gz') or \
+       (data_src == './datasrc/ENADE_2008.csv.gz') or (data_src == './datasrc/ENADE_2009.csv.gz'):
+        cols = ['nu_ano', 'co_ies', 'co_grupo', 'co_curso',
+                'co_uf_habil', 'co_munic_habil', 'nu_idade', 'tp_sexo',
+                'in_grad', 'tp_pres', 'nt_ger', 'nt_fg', 'nt_ce']
 
     df = pd.read_csv(data_src, compression=compress, dtype=ENADE_DTYPE,
                      sep=';', decimal=decimal, usecols=cols)
-
+    
+    # Convert name of columns to uppercase			
+    df.columns = map(str.upper, df.columns)
+    
+    # Standardized all columns in every years 
+    if df['NU_ANO'][0] == 2004 or 2005 or 2006 or 2007 or 2008 or 2009:
+        # Rename columns
+        df.rename(columns = {'IN_GRAD':'TP_INSCRICAO',
+                             'CO_UF_HABIL':'CO_UF_CURSO', 
+                             'CO_MUNIC_HABIL':'CO_MUNIC_CURSO'}, inplace = True)
+
+        # Between 2004 and 2009 only "modalidade presencial"
+        df['CO_MODALIDADE'] = np.ones(len(df))
```

Comment:
Is it true that all courses prior to 2010 were presential? Consider removing this column if it is not used instead of injecting data that may be wrong